Description: simply spawn 'man marked' instead of trying to locate marked.1
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2012-03-15
--- a/bin/marked
+++ b/bin/marked
@@ -24,7 +24,7 @@
   };
 
   spawn('man',
-    [__dirname + '/../man/marked.1'],
+    ['marked'],
     options);
 }
 
